packages/vanilla/src/sass/components/banner/_info.scss

```diff
@@ -2,18 +2,21 @@
 @use 'mixin' as *;
 
 .cbp-banner__info {
+
   @include banner-info(var(--cbp-color-info-dark)) {
+
     & > div > i {
+      display: flex;
       align-items: center;
+      flex-shrink: 0;
+      color: var(--cbp-color-info-darker);
       background-color: var(--cbp-color-info-light);
       border-radius: 50%;
-      color: var(--cbp-color-info-darker);
-      display: flex;
-      font-size: 24px;
-      height: 48px;
+      font-size: var(--cbp-font-size-masthead-2);
```

Comment:
This is actually the icon size. I replaced this with the space token and added a comment about it. It will be refactored to height/width when we move to SVGs.